frontend/.vscode/settings.json

```diff
@@ -1,17 +1,16 @@
 {
-  // TODO uncomment once we enable new eslint & prettier support across the entire project
-  // "[javascript]": {
-  //   "editor.formatOnSave": true
-  // },
-  // "[javascriptreact]": {
-  //   "editor.formatOnSave": true
-  // },
-  // "[typescript]": {
-  //   "editor.formatOnSave": true
-  // },
-  // "[typescriptreact]": {
-  //   "editor.formatOnSave": true
-  // },
+  "[javascript]": {
+    "editor.formatOnSave": true
```

Comment:
@christianvogt fyi, i went ahead and uncommented these lines to always enable prettier